### ISSUE TITLE
[appium-service]Error:  Cannot read property 'toLowerCase' of undefined

### ISSUE THREAD

**Issue by sanlengjingvv** [webdriverio/webdriverio#5236]
**Environment (please complete the following information):**
 - **WebdriverIO version:*6.0.14*
 - **Mode:*WDIO Testrunner*
 - **If WDIO Testrunner, running sync/async:*sync*
 - **Node.js version:*v12.16.2*
 - **NPM version:*6.14.4*
 - **Platform name and version:*MacOS 10.15.1*
 - **Additional wdio packages used (if applicable):*@wdio/appium-service 6.0.12*

**Config of WebdriverIO**
```javascript
    services: [
        ['appium', {}]
    ],
```
**Describe the bug**
execute `npx wdio wdio.conf.js` failed

**Log**
> Execution of 1 spec files started at 2020-04-09T08:34:06.863Z
2020-04-09T08:34:06.865Z DEBUG @wdio/utils:initialiseServices: initialise service "appium" as NPM package
2020-04-09T08:34:06.887Z INFO @wdio/cli:launcher: Run onPrepare hook
anyproxy ready
2020-04-09T08:34:06.893Z DEBUG @wdio/appium-service: Will spawn Appium process: node /Users/niconi/Documents/workspace/GradationAir/node_modules/appium/build/lib/main.js --base-path / --address 127.0.0.1 --command-timeout 7200 --session-override --debug-log-spacing --show-ios-log --native-instruments-lib --isolate-sim-device
2020-04-09T08:34:07.416Z DEBUG @wdio/appium-service: Appium started with ID: 45821
2020-04-09T08:34:07.417Z INFO @wdio/cli:launcher: Run onWorkerStart hook
2020-04-09T08:34:07.418Z INFO @wdio/local-runner: Start worker 0-0 with arg: run,wdio.conf.js,--spec,./tools/iOS-install-mobileprovision.js
[0-0] 2020-04-09T08:34:07.684Z INFO @wdio/local-runner: Run worker command: run
[0-0] 2020-04-09T08:34:08.267Z DEBUG @wdio/local-runner:utils: init remote session
[0-0] 2020-04-09T08:34:08.270Z INFO webdriverio: Initiate new session using the ./protocol-stub protocol
[0-0] RUNNING in undefined - /tools/iOS-install-mobileprovision.js
[0-0] 2020-04-09T08:34:08.359Z DEBUG @wdio/utils:initialiseServices: initialise service "appium" as NPM package
[0-0] 2020-04-09T08:34:08.363Z DEBUG @wdio/local-runner:utils: init remote session
[0-0] 2020-04-09T08:34:08.373Z INFO webdriverio: Initiate new session using the devtools protocol
[0-0] 2020-04-09T08:34:08.374Z ERROR @wdio/runner: TypeError: Cannot read property 'toLowerCase' of undefined
    at launch (/Users/niconi/Documents/workspace/GradationAir/node_modules/devtools/build/launcher.js:81:48)
    at Function.newSession (/Users/niconi/Documents/workspace/GradationAir/node_modules/devtools/build/index.js:55:49)
    at remote (/Users/niconi/Documents/workspace/GradationAir/node_modules/webdriverio/build/index.js:58:41)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at async Runner._startSession (/Users/niconi/Documents/workspace/GradationAir/node_modules/@wdio/runner/build/index.js:212:50)
    at async Runner._initSession (/Users/niconi/Documents/workspace/GradationAir/node_modules/@wdio/runner/build/index.js:181:21)
    at async Runner.run (/Users/niconi/Documents/workspace/GradationAir/node_modules/@wdio/runner/build/index.js:99:15)
[0-0]  Error:  Cannot read property 'toLowerCase' of undefined
2020-04-09T08:34:08.492Z DEBUG @wdio/local-runner: Runner 0-0 finished with exit code 1
[0-0] FAILED in undefined - /tools/iOS-install-mobileprovision.js
2020-04-09T08:34:08.493Z INFO @wdio/cli:launcher: Run onComplete hook
2020-04-09T08:34:08.493Z DEBUG @wdio/appium-service: Appium (pid: 45819) killed
Spec Files:      0 passed, 1 failed, 1 total (100% completed) in 00:00:01

**Comment by christian-bromann** [comment]
fixed by #5238

In the meantime please set `hostname: "localhost"` in your config to properly connected to the started Appium server.

**Comment by sanlengjingvv** [comment]
I found that the "port" parameter was required.
The entire wdio.conf.js could  reproduce this problem.
```javascript
exports.config = {
    runner: 'local',
    // port: 4723,
    maxInstances: 1,
    capabilities: [
        {
            'appium:platformName': 'iOS',
            'appium:platformVersion': '13.3.1',
            'appium:deviceName': 'deviceName',
            'appium:udid': 'udid'
        }
    ],
    baseUrl: '/',
    services: [
        ['appium', {
            args: {
            }
        }]
    ],
    framework: 'mocha',
    reporters: ['spec']
}
```
